dc

```diff
@@ -10,6 +10,24 @@ if [ -f .env ]; then
   IFS=$'\n'
   for x in $(grep -v '^#.*' .env); do export $x; done
   IFS=$OLD_IFS
+else
+  # ensure a .env is present
+  touch .env
+fi
```

Comment:
```suggestion
fi
```

You don't need that, if .env doesn't exist the code that does `>> .env` will create the file